### ISSUE TITLE
Can't see node containers

### ISSUE THREAD

**Issue by xgodon** [shipyard/shipyard#793]
Hello,

I run a shipyard node who communicates with the main shipyard via a firewall.
I added these permissive rules :
- server to node:2375
- node to server:2375
- node to server:4001
- node to server:7001

But i still can't see my containers :

![capture](https://cloud.githubusercontent.com/assets/9975606/14431918/234c5302-0008-11e6-8889-73202a78cfc1.JPG)

Could you help me resolve this? Or at least show me where i can find what ports i have to open?
Sorry for my bad english, and sorry if i missed some documentation.

Thanks!

**Comment by ehazlett** [comment]
You should be able to check the logs of the firewall and see what is getting blocked.

**Comment by xavier-godon** [comment]
Sadly i can't, it's on a cloud, and i have no access to the logs. Moreover, the hosting society isn't very efficient... Well i'll ask but i dubt i'll have the logs.

**Comment by meepeek** [comment]
I am using local machine and I got the same problem. I saw a healthy node and the number of containers within them but cannot see the container in the containers tab

As the following images, there should be node thee-166 in the list. I tried adding reserved CPU more than the primary host have and it cannot be created. I expected it will create the container in child node.

![screen shot 2559-05-27 at 10 50 57 am](https://cloud.githubusercontent.com/assets/2456033/15597324/2b520488-23f9-11e6-9543-1a7ff6d3b7d4.png)

![screen shot 2559-05-27 at 10 51 18 am](https://cloud.githubusercontent.com/assets/2456033/15597332/3ad0cc14-23f9-11e6-9175-685272de5ec7.png)
